go-controller/pkg/ovn/ovn.go

```diff
@@ -256,11 +256,21 @@ func (oc *Controller) Run() error {
 	}
 
 	for _, f := range []func() error{oc.WatchNamespaces, oc.WatchPods, oc.WatchServices,
-		oc.WatchEndpoints, oc.WatchNetworkPolicy, oc.WatchEgressNodes, oc.WatchEgressIP} {
+		oc.WatchEndpoints, oc.WatchNetworkPolicy} {
```

Comment:
it would be clearer to rebase this patch earlier so you can write that code correctly the first time

Reply:
I did it like this for easier review. I can squash those commits though